.circleci/config.yml

```diff
@@ -1383,9 +1460,11 @@ workflows:
       - spm-release-build
       - spm-release-build-xcode-14
       - spm-release-build-xcode-15
+      - spm-release-build-xcode-16-1
```

Comment:
oh interesting, you're running on the beta on purpose? or should we run this on xcode 16 instead?

Reply:
I think @tonidero made this change - can you think of a reason not to do 16.0 @tonidero ?

Reply:
Oh I think the `spm-release-build` would run in xcode 16.0 with the other changes?